benchmark/cluster/README.md

```diff
@@ -0,0 +1,54 @@
+# Cluster Training Benchmark
+
+## Setup
+
+- Platform
+  - Kubernetes: v1.6.2
+  - Linux Kernel: v3.10.0
+
+- Resource
+  - CPU: 10 Cores per Pod
+  - Memory: 5GB per Pod
+
+- Docker Image
+
+  We use different base Docker Image to run the benchmark on Kubernetes:
+  - PaddlePaddle v2: paddlepaddle/paddle:latest
+  - PaddlePaddle Fluid: paddlepaddle/paddle:latest
+  - TensorFlow: tensorflow/tensorflow:latest
+
+- Model
+  A digits recognize model and MNIST dataset is used in this benchmark.
+
+## Compare the Performance
+
+- Variable
+  - Batch Size of training data.
+  - PServer count of the training job.
+
+- Invariant
+  - The number of trainers.
```

Comment:
What is the trainer count we plan to try?

Reply:
Done. 
And @typhoonzero reminds me that we need to measure the parallel efficiency by increasing the trainer count.